orm/test/runtime/org.jboss.tools.hibernate.runtime.v_4_3.test/META-INF/persistence.xml

```diff
@@ -0,0 +1,18 @@
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"

```

Comment:
Same here, this file should be removed, no?

Reply:
Without the file, this test fails:
```
org.jboss.tools.hibernate.runtime.spi.HibernateException: Persistence unit not found: 'test'.
	at org.jboss.tools.hibernate.runtime.v_4_3.internal.ServiceImplTest.testNewJpaConfiguration(ServiceImplTest.java:137)
```
It's thrown from `ServiceImpl.newJpaConfiguration(...)` in 4_3 version.  In 4_0 version, the method is different and doesn't  throw that exception.

In GIT I can't tell why these versions are different...so not sure where to fix this...

Reply:
Are you sure? On my box the tests run fine without this persistence.xml file... In fact ServiceImplTest#testNewJpaConfiguration creates the persistence.xml file and removes it after the test run. If you run the tests than you should see that the test erased this file even.